src/set_theory/cardinal/basic.lean

```diff
@@ -593,7 +599,7 @@ begin
           ... ≤ #β          : (f.option_elim b hb).cardinal_le
 end
 
-lemma succ_pos (c : cardinal) : 0 < succ c := by simp
+lemma succ_pos (c : cardinal) : 0 < succ c := by simp [lt_succ_iff]
```

Comment:
Can you prove this with [`order.bot_lt_succ`](https://leanprover-community.github.io/mathlib_docs/order/succ_pred/basic.html#order.bot_lt_succ)`?